src/PrestaShopBundle/Resources/config/routing/admin/improve/design/mail_theme.yml

```diff
@@ -1,3 +1,7 @@
+_mail_theme_alias:
+  resource: "_mail_theme_alias.yml"
```

Comment:
@jolelievre Is it alias or may be deprecated routes ?

Reply:
well these are aliases for deprecated routes 😅
do you prefer me to rename the file into `_mail_theme_deprecated.yml` ?

Reply:
lol I prefer because aliases doesn't sound deprecated.